examples/qec_decoder_toolkit/Quantinuum_hseries_qec_decoder_toolkit.ipynb

```diff
@@ -0,0 +1,1166 @@
+    "    circuit.X(qreg[i], condition_bits=[pfu_old[i]], condition_value=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Three measurement operations are applied to qubits within `qreg`. The outcome of the measurements is stored in the corresponding bits within `creg`, i.e. a measurement on `qreg[0]` will have a measurement outcome stored in `creg[0]`. Three Measurement operations are added to each qubit in `qreg` and the corresponding bit in `creg`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in range(3):\n",
+    "    circuit.Measure(qreg[i], creg[i]);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`add_classicalexpbox_bit`, `add_c_setbits`, `add_classicalexpbox_register` are used to perform additional operations on the classical registers. The `^` operation is also used instead of `RegXOR` to perform `XOR` on two classical registers. `add_classicalexpbox_bit` is used to perform operations on specific bits, whilst `add_classicalexpbox_register` allows classical operations on entire bit registers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit.add_classicalexpbox_bit(creg[0] ^ creg[1], [syn[0]])\n",
+    "circuit.add_classicalexpbox_bit(creg[1] ^ creg[2], [syn[1]]);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit.add_classicalexpbox_register(syn_old ^ syn, syn_new);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit.add_wasm_to_reg(\"decode3\", wasm_file_handler, [syn_new, pfu], [pfu_old])\n",
+    "for i in range(3):\n",
+    "    circuit.X(qreg[i], condition_bits=[pfu_old[i]], condition_value=1);"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "circuit.add_c_setreg(False, pfu)\n",
+    "circuit.add_classicalexpbox_register(pfu_old ^ creg, pfu_old);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The repetition code circuit with the Wasm operation can be submitted to the `H2-1E` emulator using the usual workflow: compilation, costing, submission, status check and, finally, job retrieval. The linked Wasm module must be specified during job submission."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "c:\\Users\\Irfan.Khan\\.conda\\envs\\pytket-quantinuum-env\\lib\\site-packages\\requests\\__init__.py:86: RequestsDependencyWarning: Unable to find acceptable character detection dependency (chardet or charset_normalizer).\n",
```

Comment:
Maybe you want to remove this?

Reply:
resolved by upgrading requests, chardet, urllib3 to latest version